integration-tests/tests/login_test.go

```diff
@@ -174,3 +180,27 @@ func cmdInteract(f *os.File, prompt, input string) error {
 
 	return nil
 }
+
+type storeSuite struct {
```

Comment:
I suggest not using that sort of object inheritance simulation, as it's very error prone and doesn't look so great either. Instead, define a type that does exactly what you need, with the interface you need, and use it where appropriate. For example, writeCredentials doesn't even have to be on a suite. It can easily be a top-level `func authenticate(...)` definition. Then, having a new suite just to store username and password fields in one test seems like overdoing it too. Just pick up the env values in that one test.